server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts

```diff
@@ -1,9 +1,23 @@
 import { Router, Request, Response } from 'express';
+import { graphqlRequest } from '../../GraphqlHTTPRequest';
+import { GET_EXPOSURE_INFO } from '../../DBService/Exposure/Query';
+import { UPDATE_EXPOSURE, CREATE_EXPOSURE } from '../../DBService/Exposure/Mutation';
 
-const exosureRoute = Router();
+const exposureRoute = Router();
 
-exosureRoute.get('/', (request: Request, response: Response) => {
-    response.send('Hello from Exposure route');
+exposureRoute.get('/:investigationId', (request: Request, response: Response) => {
```

Comment:
u sure u dont get 404, last time I did it that way I got 404
so I had to change to that:

![image](https://user-images.githubusercontent.com/52863381/93014469-f36d4180-f5b9-11ea-9a87-2071e2281121.png)

Reply:
its working, i dont get 404, 
<img width="588" alt="Screen Shot 2020-09-13 at 12 40 13" src="https://user-images.githubusercontent.com/24268917/93015000-506af680-f5be-11ea-83a6-644de3b11869.png">